Makefile

```diff
@@ -44,7 +44,7 @@ help:
 
 up: export DEBUG = 'knex:query'
 
-up: require create-if-missing.env tmp-downloads/owid_metadata.sql.gz lerna
+up: require create-if-missing.env tmp-downloads/owid_metadata.sql.gz
```

Comment:
All the tasks that depended on `lerna` should now depend on `node_modules` (which Lerna depended on).